frontend/src/components/Search.js

```diff
@@ -65,29 +64,19 @@ export default class Search extends Component {
           <span>Search Results</span>
           <i id="spinner" className="fas fa-circle-notch fa-spin"></i>
         </div>
-        <Droppable droppableId="search-results" isDropDisabled={true}>
```

Comment:
We no longer have one Droppable (used by react-beautiful-dnd to indicate areas that "draggables" can be "dropped" in) for the entire search results container. Instead, each SearchCourseCard is a droppable.